pkg/gatherers/clusterconfig/gather_cluster_ingress_test.go

```diff
@@ -0,0 +1,51 @@
+package clusterconfig
+
+import (
+	"context"
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configfake "github.com/openshift/client-go/config/clientset/versioned/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_gatherClusterIngress(t *testing.T) {
+	tests := []struct {
+		name         string
+		config       *configv1.Ingress
```

Comment:
Calling the attribute `config` doesn't make much sense IMHO. `ingress` or `ingressDefinition` would be better IMHO.